solr/core/src/java/org/apache/solr/handler/ContentStreamHandlerBase.java

```diff
@@ -63,6 +70,17 @@ public void handleRequestBody(SolrQueryRequest req, SolrQueryResponse rsp) throw
           SolrException.ErrorCode.SERVER_ERROR,
           "Updates are temporarily paused for core: " + req.getCore().getName());
     }
+    List<CircuitBreaker> trippedCircuitBreakers =
+        req.getCore().getCircuitBreakerRegistry().checkTripped(RequestType.UPDATE);
```

Comment:
Noting that in `SearchHandler` there is a `if (circuitBreakerRegistry.isEnabled())` guard -- wondering if it might be helpful here too?